.github/workflows/e2e.yaml

```diff
@@ -27,23 +27,32 @@ jobs:
     - name: Create virtual environment
       run: make venv
 
-    - name: Cleanup existing environment
+    - name: Delete clusters
       if: ${{ always() }}
+      working-directory: test
       run: |
-        source venv
-        make destroy-rdr-env
+        source ../venv
+        drenv delete --name-prefix ${{ env.NAME_PREFIX }} envs/regional-dr.yaml
```

Comment:
Please use the make target, we will extend the scripts in to do other things in the future. The make target is running the same drenv command as you have used.

Reply:
The make target is bad for CI - we need access to drenv parameters to so we can tweak the CI run when needed.  The make target is also not good way to run locally since you need a way to add debug logs (-v) or other options. I don't mind keeping the target for as an easy way (like make coverage).